tasks/main.yml

```diff
@@ -14,11 +14,16 @@
 
 - name: Ensure Cockpit Web Console packages are installed.
   package:
-    name: "{{ __cockpit_packages[cockpit_packages] }}"
+    name: "{{ __cockpit_packages[cockpit_packages]
+              if cockpit_packages in __cockpit_package_types
+              else cockpit_packages }}"
```

Comment:
@tabowling This does the conditional stuff - if user sets `cockpit_packages` to one of `minimal`, `default`, or `full`, then install the list of packages for that group defined for the platform in `vars/PLATFORM.yml` - otherwise, assume `cockpit_packages` contains the package(s) to install.

Reply:
That looks perfect!  Thank you.